lib/gitlab/client/projects.rb

```diff
@@ -471,5 +471,23 @@ def star_project(id)
     def unstar_project(id)
       delete("/projects/#{url_encode id}/star")
     end
+
+    # Get projects by user
+    #
+    # @example
+    #   Gitlab.user_projects(1)
+    #   Gitlab.user_projects(1, { order_by: 'last_activity_at' })
+    #   Gitlab.user_projects(1, { order_by: 'name', sort: 'asc' })
+    #
+    # @param  [Integer] user_id The ID of a user.
```

Comment:
According to the doc this can also be the username, could you update the description?